### PR TITLE
WIP Brutal AI -> Extreme AI

### PR DESCRIPTION
**Pull request by CnCRAZER**
Inspired by the Mental Omega AI Boost and also within the limits of no engine extensions, these values seem to replicate a similar sense of difficulty without causing massive buildup of units and extreme lag.

Buildings have been added for AI to 'clone' vehicles during production and mostly just [AI] and [General] values have been changed. 

These values can be tweaked and adjusted as we see fit if any issues arise of course. 

I believe I have properly renamed any file containing a line with 'Brutal' to 'Extreme' and perhaps in the future could create a series of AITriggerTypes within this ini file which could apply on any map to further enhance the 'Extreme AI'.

### CODE REVIEW

File: package/updateexec [hunk 1]
```diff
@@ -451,4 +451,4 @@ Maps\Yuri's Revenge\Survival\6_survival_of_horizon_sota_ai.png
 [DeleteFolder]
 Maps\Yuri's Revenge\CTF
 Maps\Yuri's Revenge\Demolition
-do_not_remove_this_line
\ No newline at end of file
+do_not_remove_this_line
```
Comment (pzhlkj6612) [inline package/updateexec]: Hi! Does this change affect something about the updater? I don't know the updater, so this is just a question.

Comment (CnCRAZER) [reply package/updateexec]: I think I opened this file and saved it without making any changes on accident which is why it's showing up. Nothings changed

**Comment by CnCRAZER** [comment]
Global values now included to override aimd.ini

Removed secondary barracks for AI production as it wasn't very necessary + couple other things

**Comment by CnCRAZER** [comment]
The last two weeks or so of injecting this code into regular .map files and not selecting brutal ai in the client have had really positive results for all players participating. 

I believe it is ready to be merged now.

**Pull request closed without merging**